### PR TITLE
chore: Use `-tags 'netgo'` in bulding process to avoid SIGSEGV because of the different version of glibc in dfferent Linux distros

### LINKED ISSUE #604: based arch linux dist running ecapture crashed: segmentation violation
**Describe the bug**
最近的两个版本，0.8.4 和 0.8.5 在 arch 中运行都会出现一样的 `SIGSEGV: segmentation violation`
不一定总是在 Module.run() 出现问题，在 `https server starting...You can update the configuration file via the HTTP interface.` 与 `BPF bytecode file is matched. bpfFileName=user/bytecode/openssl_3_0_0_kern_core.o`
但是最常出现的就是前两个输出后立刻 segmentation violation

**To Reproduce**
1. download ecapture from releases page
2. `# ./ecapture [any command]`
3. starting

**Expected behavior**
正常捕获内容

**Screenshots**
https://fars.ee/b9La
[V.log](https://github.com/user-attachments/files/16738851/V.log)
https://fars.ee/gU9L
[V2.log](https://github.com/user-attachments/files/16738852/V2.log)
https://fars.ee/_SWJ
[v3.log](https://github.com/user-attachments/files/16738853/v3.log)
[V3.log](https://github.com/user-attachments/files/16738856/V3.log)

**Linux Server/Android (please complete the following information):**

- Env:
```
---------------------------------------
eCapture Makefile Environment:
---------------------------------------
PARALLEL                 4
----------------[ from args ]---------------
CROSS_ARCH               
ANDROID                  0
DEBUG                    0
SNAPSHOT_VERSION         
---------------------------------------
HOST_ARCH                x86_64
UNAME_R                  6.10.6-zen1-1-zen
CLANG_VERSION            18
GO_VERSION               1.23
---------------------------------------
CMD_CLANG                clang
CMD_GIT                  git
CMD_GO                   go
CMD_INSTALL              install
CMD_LLC                  llc
CMD_MD5                  md5sum
CMD_PKGCONFIG            pkg-config
CMD_STRIP                llvm-strip
CMD_CC_PREFIX            
CMD_TAR                  tar
CMD_RPMBUILD             rpmbuild
CMD_RPM_SETUP_TREE       rpmdev-setuptree
---------------------------------------
VERSION_NUM              0.8.5-20240819-35cc757
LAST_GIT_TAG             0.8.5-20240819-35cc757
BPF_NOCORE_TAG           6_10_6-zen1-1-zen.0_8_5-20240819-35cc757
KERN_RELEASE             6.10.6-zen1-1-zen
KERN_BUILD_PATH          /lib/modules/6.10.6-zen1-1-zen/build
KERN_SRC_PATH            /lib/modules/6.10.6-zen1-1-zen/build
TARGET_ARCH              x86_64
GOARCH                   amd64
LINUX_ARCH               x86
LIBPCAP_ARCH             x86_64-pc-linux-gnu
AUTOGENCMD               test -f kern/bpf/x86/vmlinux.h || bpftool btf dump file /sys/kernel/btf/vmlinux format c > kern/bpf/x86/vmlinux.h
PACKAGE_VERSION          0.0.0
OUT_DEB_FILE             ./bin/ecapture_v0.0.0_linux_amd64.deb
--------------------------------------

```
- OS: arch
- Arch: amd64/x86-64
- Kernel Version: in Env
- Version: linux_amd64:v0.8.5:6.5.0-1025-azure

**Additional context**
在 #581 提及过一次

### PR DESCRIPTION
**Pull request by Zheaoli**
Fix #604

Signed-off-by: Manjusaka <me@manjusaka.me>

### CODE REVIEW

File: functions.mk [hunk 1]
```diff
@@ -49,7 +49,7 @@ define gobuild
 	CGO_CFLAGS='-O2 -g -gdwarf-4 -I$(CURDIR)/lib/libpcap/' \
 	CGO_LDFLAGS='-O2 -g -L$(CURDIR)/lib/libpcap/ -lpcap -static' \
 	GOOS=linux GOARCH=$(GOARCH) CC=$(CMD_CC_PREFIX)$(CMD_CC) \
-	$(CMD_GO) build -tags $(TARGET_TAG) -ldflags "-w -s -X 'github.com/gojue/ecapture/cli/cmd.GitVersion=$(TARGET_TAG)_$(GOARCH):$(VERSION_NUM):$(VERSION_FLAG)' -linkmode=external -extldflags -static " -o $(OUT_BIN)
+	$(CMD_GO) build -tags '$(TARGET_TAG) netgo' -ldflags "-w -s -X 'github.com/gojue/ecapture/cli/cmd.GitVersion=$(TARGET_TAG)_$(GOARCH):$(VERSION_NUM):$(VERSION_FLAG)' -linkmode=external -extldflags -static " -o $(OUT_BIN)
```
Comment (cfc4n) [inline functions.mk:52]: In the new version of Golang, should multiple tags be separated by commas?

```shell
go help build

-tags tag,list
		a comma-separated list of additional build tags to consider satisfied
		during the build. For more information about build tags, see
		'go help buildconstraint'. (Earlier versions of Go used a
		space-separated list, and that form is deprecated but still recognized.)
```

**Review by cfc4n — APPROVED** [review]

**Review by cfc4n — CHANGES_REQUESTED** [review]
`tags` is a comma-separated list.

**Comment by Zheaoli** [comment]
Fixed, `comma` should be better

**Review by cfc4n — APPROVED** [review]
LGTM, thanks.

**Pull request merged**